package.json

```diff
@@ -43,6 +43,7 @@
     "prismjs": "^1.29.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "swiper": "^5.4.1",
```

Comment:
It doesn't seem that this dependency is used, why do we need that?

Reply:
`react-id-swiper` is a wrapper lib for a `swiper` and as I understood from issues and documentation the `swiper` dependency is needed for some features to work, without this dependency dots won’t work, You can try it locally